docs/tutorials/average_optimizers_callback.ipynb

```diff
@@ -76,9 +76,9 @@
         "\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    stocastic_avg_sgd = tfa.optimizers.SWA(sgd, 1000, 50)
</pre><p>How is this inducing the LR schedule we talked about in the original issue? For more details, refer to the <a href="https://pytorch.org/blog/stochastic-weight-averaging-in-pytorch/" target="_blank">Stochastic Weight Averaging</a> section of this blog post. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/addons/pull/2497/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>Thanks, I'll have a look.</p>